### ISSUE TITLE
Flaky tests in hercules

### ISSUE THREAD

**Issue by koxu1996** [cspr-rad/kairos#176]
It seems that tests are failing in Hercules randomly.

**Comment by marijanp** [comment]
Could you provide a list which ones are flaky?
Also note that our CI machine is not very powerful, pushing to and updating multiple PR's will cause the system load to go above 300% which could cause some runtime delays with tests that spawn a casper network and await only a limited time for results until they fail/

**Comment by koxu1996** [comment]
E2E tests are sometimes failing with `Success key not found in JSON`.

For example [job #2179](https://hercules-ci.com/accounts/github/cspr-rad/derivations/%2Fnix%2Fstore%2Fmb7l5535bwzgv1v38z5lq8050qin8id4-vm-test-run-kairos-e2e-test.drv/log?via-job=56114940-bfb5-4268-8269-123247cef2b3) (commit 2eda67a59d3ca1210749711493cc4639220bb8ae) failed first, then succeded when I trigerred "re-run effects and failures":

![image](https://github.com/user-attachments/assets/8a0f0615-5b61-49f7-ab9a-3f11dbca0c65)

**Comment by marijanp** [comment]
Yeah, if you scroll up in the logs you will see that the cctl service didn't start properly. cctl is a oneshot and this https://github.com/cspr-rad/kairos/blob/95190703e048bcf03ed2c80912b400d7460ca072/kairos-test-utils/src/cctl.rs#L153 check will make it fail to start. This is due to the high demand of ressources especially if there are many builds running at the moment (We have a notification stream on zulip which reports the current system load of the CI machine).

The solution is to increase that timeout that was linkem, or reduce the amount of concurrent builds.
